modules/synacormediaBidAdapter.js

```diff
@@ -99,8 +110,14 @@ export const spec = {
       };
     }
   },
+
+  setValidVideoParams: function (sourceObj, destObj) {
+    Object.keys(sourceObj)
+      .filter(param => includes(VIDEO_PARAMS, param) && sourceObj[param] !== null && (!isNaN(parseInt(sourceObj[param], 10)) || !sourceObj[param].length < 1))
```

Comment:
The following line:
`!sourceObj[param].length < 1`, is type coercing  to either: `true < 1` or `false < 1` 

Should probably be:
`!(sourceObj[param].length < 1)`

Reply:
Updated PR with that change, good catch!